partner-onboarder/install.sh

```diff
@@ -74,7 +74,35 @@ function installing_onboarder() {
     --set onboarding.configmaps.s3.s3-bucket-name="$s3_bucket" \
     $ENABLE_INSECURE \
     -f values.yaml \
-    --version $CHART_VERSION
+    --version $CHART_VERSION &
+    helm_install_pid=$!
+
+    # Wait for the Helm installation command to finish
+    wait $helm_install_pid
+
+    # Check the status of the Helm installation job
+    if [ $? -eq 0 ]; then
+      # Helm installation command completed successfully, now check job status
+      kubectl wait --for=condition=complete job/mimoto-keybinding-partner-onboarder-mimoto-keybinding -n $NS --timeout=5m
+
+      # Check the status of the Helm installation job
+      if [ $? -eq 0 ]; then
+        wallet_binding_partner_api_key=$(kubectl logs -n $NS job/mimoto-keybinding-partner-onboarder-mimoto-keybinding | grep "mpartner default mimoto keybinding apikey:" | awk '{print $6}')
+        echo Wallet Binding Partner Api Key is $wallet_binding_partner_api_key
+        kubectl -n $NS create secret generic mimoto-wallet-binding-partner-api-key --from-literal=mimoto-wallet-binding-partner-api-key=$wallet_binding_partner_api_key --dry-run=client -o yaml | kubectl apply -f -
+        ./copy_cm_func.sh secret mimoto-wallet-binding-partner-api-key mimoto config-server
+        kubectl -n config-server set env --keys=mimoto-wallet-binding-partner-api-key --from secret/mimoto-wallet-binding-partner-api-key deployment/config-server --prefix=SPRING_CLOUD_CONFIG_SERVER_OVERRIDES_
+        kubectl -n config-server get deploy -o name |  xargs -n1 -t  kubectl -n config-server rollout status
+        kubectl rollout restart deployment -n mimoto mimoto
+        echo Mimoto wallet binding partner api key updated successfully.
+      else
```

Comment:
In general practive are we asking this if helm is failing?
Lets follow the standard practice.